src/transformers/models/table_transformer/configuration_table_transformer.py

```diff
@@ -237,6 +237,17 @@ def num_attention_heads(self) -> int:
     def hidden_size(self) -> int:
         return self.d_model
 
+    def to_dict(self) -> Dict[str, any]:
+        """
+        Serializes this instance to a Python dictionary. Override the default [`~PretrainedConfig.to_dict`]. Returns:
+            `Dict[str, any]`: Dictionary of all the attributes that make up this configuration instance,
+        """
+        output = copy.deepcopy(self.__dict__)
+        if output["backbone_config"] is not None:
+            output["backbone_config"] = self.backbone_config.to_dict()
+        output["model_type"] = self.__class__.model_type
+        return output
+
```

Comment:
Could you remind me why this change is necessary to allow using smaller model? Thanks
(I am OK with the change, just want to know a bit the detail)

Reply:
This is something that should have been done when adding the model. You can't serialize the config otherwise (so you can't print it for instance) when it has a `backbone_config`.